medicines/doc-index-updater/src/document_manager/mod.rs

```diff
@@ -27,9 +47,14 @@ async fn del_document_handler(
         let duration = Duration::days(1);
 
         match queue.send(message, duration).await {
-            Ok(_) => Ok(warp::reply::json(
-                &state_manager.set_status(id, JobStatus::Accepted).await?,
-            )),
+            Ok(_) => match choose_reply_content_type(accept_header) {
+                ReplyType::Xml => Ok(warp::reply::xml(
+                    &state_manager.set_status(id, JobStatus::Accepted).await?,
```

Comment:
is it possible to return just the type of `state_manager.set_status(id, JobStatus::Accepted).await` here, and then have separate Xml/Json wrapper calling this?